dotcom-rendering/fixtures/manual/articles.ts

```diff
@@ -0,0 +1,28 @@
+import {
+	ArticlePillar,
+	ArticleSpecial,
+	ArticleDisplay,
+	ArticleDesign,
+	ArticleFormat,
+} from '@guardian/libs';
+
+export const getAllThemes = (format: {
+	display: ArticleDisplay;
+	design: ArticleDesign;
+}): Array<ArticleFormat> => {
+	return [
+		{ ...format, theme: ArticlePillar.News },
+		{ ...format, theme: ArticlePillar.Sport },
+		{ ...format, theme: ArticlePillar.Culture },
+		{ ...format, theme: ArticlePillar.Lifestyle },
+		{ ...format, theme: ArticlePillar.Opinion },
+		{ ...format, theme: ArticleSpecial.SpecialReport },
+		{ ...format, theme: ArticleSpecial.Labs },
+	];
+};
+
+export const getThemeFromFormat = (format: ArticleFormat): string => {
```

Comment:
Is there perhaps a better name for this that highlights it returns the name of the theme as a string? (If I understood this correctly!)

Reply:
Yes, this is what it does, thanks! 🙏 Changing the name, good call 🙂